site/content/docs/0.0/utilities/overflow.md

```diff
@@ -69,16 +69,16 @@ Adjust the `overflow-x` property to affect the overflow of content horizontally.
 Adjust the `overflow-y` property to affect the overflow of content vertically.
 
 <div class="bd-example d-md-flex">
-  <div class="overflow-y-auto p-3 mb-3 mb-md-0 me-md-3 w-100 border" style="max-width: 200px; max-height: 100px;">
+  <div class="overflow-y-auto p-medium mb-medium mb-md-none me-md-medium w-100 border" style="max-width: 200px; max-height: 100px;">
     <code>.overflow-y-auto</code> example on an element with set width and height dimensions.
   </div>
-  <div class="overflow-y-hidden p-3 mb-3 mb-md-0 me-md-3 w-100 border" style="max-width: 200px; max-height: 100px;">
+  <div class="overflow-y-hidden p-medium mb-medium mb-md-none me-md-medium w-100 border" style="max-width: 200px; max-height: 100px;">
     <code>.overflow-y-hidden</code> example on an element with set width and height dimensions.
   </div>
-  <div class="overflow-y-visible p-3 mb-3 mb-md-0 me-md-3 w-100 border" style="max-width: 200px; max-height: 100px;">
+  <div class="overflow-y-visible p-medium mb-medium mb-md-none me-md-medium w-100 border" style="max-width: 200px; max-height: 100px;">
```

Comment:
There's an issue with this example, as the text is not long-enough to show the feature. It's mainly because we go from 20px padding to 16px padding. Maybe we should use `tall` instead in the entire page?

Reply:
Let's wait for https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2754#discussion_r1820355076 final values as it could be fixed automatically